lib/birthdayDisplay.ml

```diff
@@ -142,17 +150,20 @@ let print_anniversary_list conf base dead_people dt liste =
         Output.print_sstring conf ")</em>")
       else (
         Output.print_string conf (txt_of conf base p);
+        (* TODO year of birth *)
         match get_death p with
-        | NotDead ->
+        | NotDead -> (
             Output.print_sstring conf " <em>";
-            (match a_ref - a with
+            Output.print_sstring conf (string_of_int a);
+            Output.print_sstring conf "</em>";
+            Output.print_sstring conf ", ";
+            match a_ref - a with
             | 0 -> Output.print_sstring conf (transl conf "birth")
             | 1 -> Output.print_sstring conf (transl conf "one year old")
             | n ->
                 Output.print_sstring conf (string_of_int n);
                 Output.print_sstring conf " ";
-                Output.print_sstring conf (transl conf "years old"));
-            Output.print_sstring conf "</em>"
+                Output.print_sstring conf (transl conf "years old"))
```

Comment:
this code looks similar too code line 29, it would be nice if it can be factorized

Reply:
done